packages.json

```
@@ -1,4 +1,16 @@
 [
+  {
+    "name": "tinyfiledialogs", 
+    "url": "https://github.com/Patitotective/tinyfiledialogs.nim", 
+    "method": "git", 
+    "tags": [
+      "dialogs", 
+      "file dialogs"
```

Tags should be one word.

```suggestion
      "file-dialogs"
```